sdk/core/src/request.rs

```diff
@@ -0,0 +1,60 @@
+use http::{HeaderMap, Method};
+use std::fmt::Debug;
+use url::Url;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Request {
+    pub url: Url,
+    pub method: Method,
+    pub headers: HeaderMap,
+    pub payload: bytes::Bytes,
```

Comment:
I'm not sure I like the name `payload` better than `body`.

Reply:
renamed!